### PR TITLE
Provides a first USAGE with first top level params

### PR DESCRIPTION
**Pull request by cjeanner**
This file will be the actual doc entrypoint, showing information about the future parameters and how to use them.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# CI Framework - Usage guide
+## Purpose
+The purpose of this repository is to provide a unified tool for various usage,
```
Comment (marios) [inline USAGE.md:3]: s/usage/uses

File: USAGE.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+from dev testing to CI infra.
+
+## Workflow integration
+In the end, the main entrypoint will be from within the
+[install_yaml](https://github.com/openstack-k8s-operators/install_yamls)
```
Comment (marios) [inline USAGE.md:8]: [install_yamls]

File: USAGE.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+repository, with new Makefile targets.
+
+The Framework will then leverage install_yamls content and generate the needed
+bits in order to deploy EDPM on the selected infrastructure.
+
+The Framework will also ensure we're able to reproduce the exact same run we
+got in CI with a serie of artifacts one may download locally, and re-run.
```
Comment (marios) [inline USAGE.md:15]: series

File: USAGE.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+## Parameters
+There are two levels of parameters we may provide:
+- top level
+- role level
+
+### Top level parameters
+The following parameters allow to set a common value for parameters that
+are shared among multiple roles:
+* `cifmw_basedir`: The basedire for all of the artifacts. Defaults to
```
Comment (marios) [inline USAGE.md:25]: base directory

**Review by marios — COMMENTED** [review]
looks good
minor nits/suggestions

**Comment by cjeanner** [comment]
Thank you @marios for the corrections :).

File: USAGE.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+pre_infra:
+    - name: My glorious hook name
+      type: playbook
+      source: foo.yml
+    - name: My glorious CRD
+      type: crd
+      source: /path/to/my/glorious.crd
+```
+In the above example, the `foo.yml` is located in
+[ci_framework/hooks/playooks](ci_framework/hooks/playooks) while `glorious.crd`
```
Comment (viroel) [inline USAGE.md:69]: typo: playbooks

**Review by marios — COMMENTED** [review]
lgtm

**Pull request merged**